platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/eventhandling/PreConsensusEventHandler.java

```diff
@@ -145,13 +138,9 @@ public void clear() {
     }
 
     /**
-     * Do pre-handle processing on the pre-consensus event and add it to the queue (q1) for handling. Events that are
-     * null or empty are discarded immediately. All other events go through pre-handle processing. Events that should
-     * not be handled pre-consensus according to {@link SwirldStateManager#discardPreConsensusEvent(EventImpl)} are not
-     * added to the queue.
+     * Prehandle application transactions, and enqueue platform transactions for prehandling.
      */
-    @Override
-    public void preConsensusEvent(final EventImpl event) {
+    public void prehandleTransactions(final EventImpl event) {
```

Comment:
Given the comment above, I disagree with the renaming of this method.

Reply:
Reverted the naming of this method (with capitialization fixed... elsewhere we are using `preconsensus` not `preConsensus`).